### PR TITLE
Update HIRING.md

### PR DESCRIPTION
**Pull request by marijnschilling**
Wrote the 'about you' a bit more loosely (I don't like generic terms like teamplayer, what do we mean by that?)

Let's write 'what you'll be doing' versions for each office separate.

Will fill in the amsterdam one after lunch!

### CODE REVIEW

File: HIRING.md [hunk 1]
```diff
@@ -52,21 +52,32 @@ The candidate will be asked to make an assigment with a 24 hour deadline. They w
 # Job description for an iOS position
 
 ### A bit about you
-- 01 Familiarity with building applications for iOS.
-- 02 Teamplayer.
-- 03 Familiarity or interest in Swift, usability and Web services.
+- 01 You love building applications for iOS.
```
Comment (elland) [inline HIRING.md]: I'm not comfortable with this language. I don't care if people love it or not.

File: HIRING.md [hunk 2]
```diff
@@ -52,21 +52,32 @@ The candidate will be asked to make an assigment with a 24 hour deadline. They w
...
+- 02 You're happy working in a team.
+- 03 Interest in Swift, usability and communicating with Web services.
```
Comment (elland) [inline HIRING.md]: That's one way to remove repetition. 👍

File: HIRING.md [hunk 3]
```diff
@@ -52,21 +52,32 @@ The candidate will be asked to make an assigment with a 24 hour deadline. They w
...
 - 04 Self-sustaining and forever curious.
 - 05 Familiarity with unit testing.
 - 06 A keen eye for good design is a bonus.
-- 07 Collaborating on open source projects is also a bonus.
+- 07 Have you collaborated on open source projects? That is also a bonus.
```
Comment (elland) [inline HIRING.md]: 👏 But maybe `Do you…`? Present instead of past.

File: HIRING.md (hunk 1 above)
Comment (marijnschilling) [reply HIRING.md]: Yeah i agree sounds a bit scary! But I do like to start with speaking directly to the reader. 

maybe 'You enjoy building applications for iOS.'

Comment (elland) [reply HIRING.md]: I'm fine with the model, just not the choice of words. `Enjoy` might work, but I liked `interested` more. 😄

Comment (3lvis) [reply HIRING.md]: @marijnschilling Love it!

Comment (3lvis) [reply HIRING.md]: I would keep it as `You love building applications for iOS.`, reflects more the personality of the team.

**Comment by 3lvis** [comment]
@marijnschilling Let's ship this!

**Comment by 3lvis** [comment]
@marijnschilling Lets ship this!

File: HIRING.md (hunk 1 above)
Comment (elland) [reply HIRING.md]: Let's not. That's neither something we require nor something we can enforce. 😅

Comment (3lvis) [reply HIRING.md]: I would be fine with requiring that. About enforcing, how would you even enforce that?

Comment (elland) [reply HIRING.md]: Then I might have to leave. 😭

Comment (3lvis) [inline HIRING.md]: Why would you do things you don't love? 🤔

Comment (Elland) [inline HIRING.md]: Is this a serious question? 😕

**Review by 3lvis — APPROVED** [review]
Approved. Haha, thanks GitHub 💰

**Comment by elland** [comment]
🍰

**Pull request merged**